code/game/machinery/autolathe.dm

```diff
@@ -244,14 +244,14 @@
 	return
 
 /obj/machinery/autolathe/RefreshParts()
-	var/tot_rating = 0
-	prod_coeff = 0
+	var/T =1.2
 	for(var/obj/item/weapon/stock_parts/matter_bin/MB in component_parts)
-		tot_rating += MB.rating
-	tot_rating *= 25000
-	materials.max_amount = tot_rating * 3
+		T += MB.rating*75000
+	materials.max_amount = T

```

Comment:
This is the same amount as before but without doing the calculations in two lines. The way coeff is counted is now the same as in R&D machines. Same thing with manipulators.